Scripts/book.js

```diff
@@ -64,26 +56,88 @@ class userInterface {
   const title = document.getElementById('title').value;
   const author = document.getElementById('author').value;
   const id = Math.floor(Math.random() * 1000) + 1;
   id.toString();
   e.preventDefault();
   const newBook = new AwesomeBooks(title, author, id);
-  userInterface.addBookToList(newBook);
-  userInterface.clearFields();
-  storage.addition(newBook);
+  AwesomeBooks.addBookToList(newBook);
+  AwesomeBooks.clearFields();
+  AwesomeBooks.addition(newBook);
 });
-
 getAppended.addEventListener('click', (e) => {
   if (e.target.classList.contains('delete')) {
     e.target.parentElement.remove();
   }
-
   const tity = parseInt(e.target.parentElement.children[2].innerText, 10);
-  storage.remove(tity);
+  AwesomeBooks.remove(tity);
 });
+const getList = document.querySelector('.list');
+const getAddNew = document.querySelector('.add_new');
+const getContact = document.querySelector('.contact');
+getList.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.querySelector('.add-block').style.visibility = 'hidden';
+  document.querySelector('.add-block').style.position = 'absolute';
+  document.querySelector('.contactsection').style.visibility = 'hidden';
+  document.querySelector('.contactsection').style.position = 'absolute';
+  document.querySelector('.list-block').style.position = 'relative';
+  document.querySelector('.list-block').style.visibility = 'visible';
+  document.querySelectorAll('header a').forEach((item) => {
+    item.classList.remove('active');
+  });
+  e.target.classList.toggle('active');
+});
+getAddNew.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.querySelector('.list-block').style.visibility = 'hidden';
+  document.querySelector('.list-block').style.position = 'absolute';
+  document.querySelector('.contactsection').style.visibility = 'hidden';
+  document.querySelector('.contactsection').style.position = 'absolute';
+  document.querySelector('.add-block').style.position = 'relative';
+  document.querySelector('.add-block').style.visibility = 'visible';
+  document.querySelectorAll('header a').forEach((item) => {
+    item.classList.remove('active');
+  });
+  e.target.classList.toggle('active');
+});
+getContact.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.querySelector('.list-block').style.visibility = 'hidden';
+  document.querySelector('.list-block').style.position = 'absolute';
+  document.querySelector('.add-block').style.visibility = 'hidden';
+  document.querySelector('.add-block').style.position = 'absolute';
+  document.querySelector('.contactsection').style.position = 'relative';
+  document.querySelector('.contactsection').style.visibility = 'visible';
+  document.querySelectorAll('header a').forEach((item) => {
+    item.classList.remove('active');
+  });
+  e.target.classList.toggle('active');
+});
+const displayDate = document.querySelector('.time');
+const realTime = () => {
+  const date = new Date();
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you make use of luxon library to create your date.

## EXAMPLE
```
import { DateTime } from 'luxon'

this.localDatetime = DateTime.local().toLocaleString(DateTime.DATETIME_FULL);
````

PS: The above is an example and does not apply directly for your code